src/Microsoft.AspNet.SignalR.Core/Configuration/DefaultConfigurationManager.cs

```diff
@@ -14,12 +14,15 @@ public class DefaultConfigurationManager : IConfigurationManager
         // if _minimumKeepAlivesPerDisconnectTimeout != 3, update the ArguementOutOfRanceExceptionMessage below
         private const int _minimumKeepAlivesPerDisconnectTimeout = 3;
 
+        internal const int DefaultMaxScaleoutMappingsPerStream = 1000000;
```

Comment:
I left it for "compatibility". I am not sure if we know what the "correct" value should be (although it seems that 50000/100000 could be sufficient)